tokio-console/src/config.rs

```diff
@@ -152,6 +162,9 @@ pub struct ColorToggles {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 struct ConfigFile {
+    target_addr: Option<String>,
+    env_filter: Option<String>,
+    retain_for: Option<String>,
```

Comment:
i would probably call this
```suggestion
    retention: Option<String>,
```
in the config file

Reply:
also, take it or leave it, but we could add a `Deserialize` implementation for `RetainFor`, and then change this to just be 
```suggestion
    retain_for: Option<RetainFor>,
```
that way, we wouldn't have to handle deserializing it as a string and then parsing the string later in the `ConfigFile::retain_for` accessor.

i would imagine the `Deserialize` impl would just call the `FromStr` impl, like this:
https://github.com/serde-rs/serde/issues/908#issuecomment-298027413

Reply:
I fixed: https://github.com/tokio-rs/console/pull/334/commits/00e31553ee118dda9d1e5773c121d542723d49d2